.github/workflows/cmake.yml

```diff
@@ -0,0 +1,38 @@
+name: CMake
```

Comment:
Please remove this file. We already have a workflow that runs CMake commands. https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/.github/workflows/awesome_workflow.yml

Reply:
Disabled Awesome CI Workflow in action in my repository!
did you say this?